buildspec.yml

```diff
@@ -3,6 +3,7 @@ version: 0.2
 env:
   parameter-store:
     REGION: "region"
+    IGNORE_AWS_PUSH: "false"
```

Comment:
Is an environment variable the best way to do this? :D Seems to work but idk if it's good style

Reply:
I'd say most common way is usually command line args. Thing I'm worried about here is if the env var is not specified it will fall back to `undefined` right? What behaviour does that cause. I would almost treat this like a `--debug` flag

Reply:
Added code for a "--debug" cmd line arg that can be placed in the Dockerfile :)